apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -75,242 +80,156 @@ export function InstructorQuestionSettings({
-                          ${resLocals.authz_data.has_course_permission_edit &&
-                          !resLocals.course.example_course
-                            ? html`
-                                <a
-                                  class="btn btn-xs btn-secondary mx-2"
-                                  href="${resLocals.urlPrefix}/question/${resLocals.question
-                                    .id}/file_edit/${infoPath}"
-                                >
-                                  <i class="fa fa-edit"></i>
-                                  <span>Edit</span>
-                                </a>
-                              `
-                            : ''}
-                        `
-                      : ''}
-                  </td>
-                </tr>
-                <tr>
-                  <th>Type</th>
-                  <td>${resLocals.question.type}</td>
-                </tr>
-                <tr>
-                  <th>Topic</th>
-                  <td>${TopicBadge(resLocals.topic)}</td>
-                </tr>
-                <tr>
-                  <th>Tags</th>
-                  <td>${TagBadgeList(resLocals.tags)}</td>
-                </tr>
-                <tr>
-                  <th>Issues</th>
-                  <td>
-                    ${renderEjs(import.meta.url, "<%- include('../partials/issueBadge') %>", {
-                      count: resLocals.open_issue_count,
-                      issueQid: resLocals.question.qid,
-                      suppressLink: resLocals.suppressLink,
-                      urlPrefix: resLocals.urlPrefix,
-                    })}
-                  </td>
-                </tr>
-                <tr>
-                  <th>Assessments</th>
-                  <td>
-                    ${resLocals.assessments
-                      ? renderEjs(import.meta.url, "<%- include('../partials/assessments') %>", {
-                          assessments: resLocals.assessments,
-                          urlPrefix: resLocals.urlPrefix,
-                        })
+                <div class="table-responsive card mb-3">
+                  <table class="table two-column-description">
+                    <tr>
+                      <th class="border-top-0">Topic</th>
+                      <td class="border-top-0">
+                        <span
+                          class="badge color-${resLocals.topic.color}"
+                          data-toggle="tooltip"
+                          data-boundary="window"
+                          data-html="true"
+                          title="${unsafeHtml(resLocals.topic.description)}"
+                        >
+                          ${resLocals.topic.name}
+                        </span>
+                      </td>
+                    </tr>
+                    <tr>
+                      <th>Tags</th>
+                      <td>${TagBadges({ tags: resLocals.tags })}</td>
+                    </tr>
```

Comment:
Calling out a change relevant to recent changes by @jonatanschroeder: since this PR was opened, the tags/topic were updated to be displayed by these components:

```js
import { TagBadgeList } from '../../components/TagBadge.html.js';
import { TopicBadge } from '../../components/TopicBadge.html.js';
```

Independently of that, Myles and I thought it'd be nice to display the description of these in a tooltip, so we built separate components just in this file (`TagBadges` and the inline topic HTML above). We should probably be consistent here; should we move the description tooltip to the new components that Jonatan introduced?

Reply:
I switched back to the "standard" components in 5fd1d23. I still think using tooltips to show additional information about tags/topics is an intriguing idea, but we can save that for later; I'd like to get this PR landed.